Changelog.md

```diff
@@ -1,5 +1,8 @@
 # FOSSA CLI Changelog
 
+## Unreleased
+- Doc only: Fixed an issue where a referenced table was missing from `analyze.md`
+
```

Comment:
```suggestion
```
Thanks for considering the changelog! We don't put doc-only changes here however 😄

Reply:
_Drat!_ 😅  

I figured better safe than sorry, but I should've trusted my gut. Thank you for the suggestion! Committed.